about.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="mobile-nav">
+      <input type="checkbox" class="nav-checkbox" id="nav-switch" />
+      <label for="nav-switch" class="nav-btn">
+        <span class="nav-icon">&nbsp;</span>
+      </label>
+      <div class="nav-bck">&nbsp;</div>
+      <nav class="nav-cont">
+        <ul class="mob-cont-link">
+          <li class="mob-single-link">
+            <a href="./index.html" class="mob-link">Home</a>
+          </li>
+          <li class="mob-single-link">
+            <a href="#" class="mob-link">Program</a>
+          </li>
+          <li class="mob-single-link"><a href="#" class="mob-link">Join</a></li>
+          <li class="mob-single-link">
+            <a href="#" class="mob-link">Sponsor</a>
+          </li>
+          <li class="mob-single-link"><a href="#" class="mob-link">News</a></li>
+          <li class="mob-single-link">
+            <a href="#" class="mob-link">CC Campaign</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Please follow closely the design guidelines, in the navbar give more margin/padding to the buttons

Yours:
![Screenshot from 2021-10-15 19-17-35 (copy)](https://user-images.githubusercontent.com/62705947/137528000-cb1f17f6-b75c-4cbe-8c79-1a0764f0dbec.png)

Original:
![Screenshot from 2021-10-15 19-19-40](https://user-images.githubusercontent.com/62705947/137528024-69976e75-0509-4066-aa32-1f9e4de36ab0.png)

Reply:
- [ ] I can't see any commits or change about this issue. Please adjust the margin/padding as requested.